### PR TITLE
Updated the readme for media

### PR DESCRIPTION
**Pull request by bbrock25**
_Why_?
Cause we were totally void of documentation

_How_?
Added migration instructions and paperclip config to the readme

_Side Effects_?
Better docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,34 @@ To verify, start the server and visit `localhost:3000/admin`. If you can login
...
+        secret_access_key: ENV.fetch("AWS_SECRET", ""),
+        s3_host_name: "s3-#{ENV.fetch('AWS_REGION', '')}.amazonaws.com"
+      }
+    }
+  ...
+  end
+```
+
+
+
```
Comment (Mrjaco12) [inline README.md]: Some consistency in the blank lines plz.

**Comment by samcdavid** [comment]
:+1:

**Comment by Mrjaco12** [comment]
:+1:

**Comment by bbrock25** [comment]
fixes #35

**Pull request merged**